src/main/java/cofh/thermaldynamics/duct/Attachment.java

```diff
@@ -88,10 +88,12 @@ public void tick(int pass) {
 	public void dropItemStack(ItemStack item) {
 
 		Cuboid6 c = getCuboid();
-		Random rand = tile.getWorld().rand;
-		Vector3 dif = c.max.copy().subtract(c.max);
```

Comment:
context: this line evaluates to a Vector3 of 0,0,0 which is then multiplied with the tile position. This caused the itemstack to always spawn @ 0,0,0